### ISSUE TITLE
New icon and GUI for the launcher (also changing name of the launcher)

### ISSUE THREAD

**Issue by YannYB** [minecraft-linux/mcpelauncher-manifest#688]
**Is your feature request related to a problem? Please describe.**
I think the icon and the launcher are a bit too outdated. The name is "Linux Minecraft launcher" and when you are on macOS it's really weird.

**Describe the solution you'd like**
 I would like to see a new icon and also the ui of the official mc launcher and a new title like "Minecraft Bedrock"

**Describe alternatives you've considered**
What about a launcher with java edition and bedrock on the same for macOS and linux (I think it's impossible)

**Additional context**
<img width="101" alt="Capture d’écran 2022-05-17 à 16 31 55" src="https://user-images.githubusercontent.com/100095100/168836532-2d5a01e3-4e98-44fc-9474-014a18fd8493.png">
(like this)
and 
![image](https://user-images.githubusercontent.com/100095100/168836837-95f85ddf-2fef-4bab-a638-0f9318a32bc5.png)
(like this)

**Comment by ChristopherHX** [comment]
I also want a new Icon, but for different reasons. The minecraft-wiki has removed the official icons, after they no longer had an agreement with mojang. ( This project never had an agreement with mojang. )
I think the bedrock texture of you icon proposal seems like an official texture of the game.

The icon is the same icon as on Android, no idea why you think it is outdated.

> The name is "Linux Minecraft launcher"

I guess I was just to lazy to update the english translation to add macOS.

**Comment by YannYB** [comment]
I understand about the icon (the icon isn’t actually official it came from a guy on reddit : https://www.reddit.com/r/Minecraft/comments/qmzysq/i_didnt_like_the_inconsistency_of_the_new/ )

**Comment by ChristopherHX** [comment]
I don't want to switch to an new icon without permission to use it, otherwise I would keep the icon.
Maybe this one, created by me. Not shure about the quality, but it is a svg so can be resized without loosing precision
![bedrock6](https://user-images.githubusercontent.com/44845461/169269407-3ee1eb16-452d-47bc-b557-8da76d7ffe08.svg)

The current icon doesn't look good in my gnome desktop, way to much pixels for that small space. This is a pseudo bedrock with 8x8 texture. You clearly see the difference to the official texture.

**Comment by wertie3020** [comment]
if you want it to lock like bedrock i think the fist is better 
but meby we can create some more idees first and then ask what ather players want

**Comment by wertie3020** [comment]
i just created one where can i uplode?

**Comment by ChristopherHX** [comment]
In a github repository, cloud or drag'n drop a zip file with the icon here

**Comment by wertie3020** [comment]
![minecrarftBedrockLogo:256x256](https://user-images.githubusercontent.com/68184807/169359488-90af4dd3-18a9-4278-8c21-bd0a8a79c931.png)

**Comment by sashakoshka** [comment]
Just made this, what do yall think?
![minecraft bedrock linux launcher](https://user-images.githubusercontent.com/83851797/169396055-9722c647-7169-4223-9789-fd92776c4f7b.svg)

**Comment by ChristopherHX** [comment]
@wertie3020 The problem I have with such a detailed icon, similar to the old one, is that my dock / taskbar has not enough pixels.

BTW I want an svg, because a png will cause me trouble with the flatpak.
[bedrock7.zip](https://github.com/minecraft-linux/mcpelauncher-manifest/files/8733937/bedrock7.zip)
![bedrock7](https://user-images.githubusercontent.com/44845461/169395701-26f6fe28-6b4d-4b4e-99cd-0246a37c6722.svg)
I fixed my icon, to close the svg gaps. The same problem had my previous svg icon for the flatpak.

This is my crypto block
![bedrock9](https://user-images.githubusercontent.com/44845461/170269353-dae27600-abb5-468b-bea1-6ea66d31f4cb.svg)
[bedrock9.zip](https://github.com/minecraft-linux/mcpelauncher-manifest/files/8771390/bedrock9.zip)


@sashakoshka Thank you your icon looks good, I'm just a bit confused to see a bedrock block with grass but thats it.
We can vote later.

**Comment by sashakoshka** [comment]
It has come to my attention that the svg didn't work in some places due to having a drop shadow. This one has no drop shadows (and also better contrast):
![minecraft bedrock](https://user-images.githubusercontent.com/83851797/169411615-d7e4561a-7114-41fc-9c99-7f2dd7389b0a.svg)

**Comment by sashakoshka** [comment]
This version has more saturated grass. I apologize for clogging up the comment section, I can't seem to get these images to display any smaller...

![Untitled 57](https://user-images.githubusercontent.com/83851797/169621207-09c6e4a7-059e-42b0-b4cd-697854b98033.svg)

**Comment by sashakoshka** [comment]
Also made a title banner. I don't know if people will understand "unix launcher" but "linux/macos launcher" seemed too wordy.
![banner](https://user-images.githubusercontent.com/83851797/169700676-d4687b81-7628-49a5-ba8f-767b6c060c3f.svg)

**Comment by sashakoshka** [comment]
Fixed some alignment and perspective issues
![banner](https://user-images.githubusercontent.com/83851797/169713171-3f23f17b-1345-416a-9f09-d91f3e243ae2.svg)

**Comment by wertie3020** [comment]
locks good to me

**Comment by ChristopherHX** [comment]
@sashakoshka I'm going to switch to your icon / title banner logo thank you for your contribution.
Here is a preview of how the next release might look like. ~~However linux is not an unix~~, maybe you could replace unix with nix? This is probably something to discuss.

![image](https://user-images.githubusercontent.com/44845461/170823786-0fea2766-7ed3-4b61-b666-1cc4f3dd6f33.png)

These icons will be added to https://github.com/minecraft-linux/mcpelauncher-ui-qt/tree/ng/Resources and the proprietary git submodule removed.

**Comment by YannYB** [comment]
> Fixed some alignment and perspective issues
> ![banner](https://user-images.githubusercontent.com/83851797/169713171-3f23f17b-1345-416a-9f09-d91f3e243ae2.svg)

I really love the logo !

**Comment by sashakoshka** [comment]
> @sashakoshka I'm going to switch to your icon / title banner logo thank you for your contribution.
> 
> Here is a preview of how the next release might look like. ~~However linux is not an unix~~, maybe you could replace unix with nix? This is probably something to discuss.
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/44845461/170823786-0fea2766-7ed3-4b61-b666-1cc4f3dd6f33.png)
> 
> 
> 
> These icons will be added to https://github.com/minecraft-linux/mcpelauncher-ui-qt/tree/ng/Resources and the proprietary git submodule removed.

That's a good idea, I didn't think of that. I can do that right now.

**Comment by sashakoshka** [comment]
![banner](https://user-images.githubusercontent.com/83851797/170830622-4b27ec3e-a9db-4a4d-816c-bf1cfc8315ef.svg)

**Comment by ChristopherHX** [comment]
New Icon / Logo is applied to the launcher.
No I don't want to write a new GUI.
